src/qt/walletmodel.cpp

```diff
@@ -704,10 +704,10 @@ bool WalletModel::hdEnabled() const
 
 int WalletModel::getDefaultConfirmTarget() const
 {
-    return nTxConfirmTarget;
+    return FIXME_IMPLEMENT_IPC_VALUE(nTxConfirmTarget);
 }
 
 bool WalletModel::getDefaultWalletRbf() const
 {
-    return fWalletRbf;
```

Comment:
I think that's not optional but a required element of making this asynchronous, otherwise there'd be a lot of roundtrips.